Dependencies/xr/Source/ARCore/XR.cpp

```diff
@@ -1313,14 +1340,12 @@ namespace xr
 
     uintptr_t System::GetNativeXrContext()
     {
-        // TODO
-        return 0;
+        return reinterpret_cast<uintptr_t>(&m_impl->XrContext);
```

Comment:
Not new, but it seems a little weird to return this as an opaque pointer here since it is NativeXR that has knowledge of napi and makes the decision to expose the opaque pointer. Would it make more sense to return an IContext and then let NativeXr choose to cast to an opaque pointer?

Reply:
we don't currently have a universal IContext definition. if we want a global interface we could pass around a pointer. Otherwise IContext likely will just be a typedef of a similar opaque pointer